README.md

```diff
@@ -1,2 +1,10 @@
 # NU17
 Steamworks robot code
+
+To Generate a Javadoc:
+1) go to Project, Generate Javadoc.
+2) Javadoc Command: C:\Program Files\Java\jdk1.8.0_111\bin\javadoc.exe
```

Comment:
This will be different for other people because it bakes in a version of the jdk